### PR TITLE
rev up those droprates

### PR DESCRIPTION
**Pull request by ghost**
I've heard mixed opinions on my buff to droprates, piggybacking on ergo's proposed buff (and subsequent nerf to rnd prices) but the opinion has mostly been 'not enough'. This is a tiny bit of an update and I'd like to hear your opinions since it's mostly value editing and really easily changed.

tl;dr: mining should be mining again and not monster hunter laggy 2d edition

(also diamond no longer spawns in the primary chunk of the east asteroid at all)

### CODE REVIEW

**Comment by Iamgoofball** [comment]
someday we need to find urist and drag his lazy ass back over here and make him finish adamantite

File: code/modules/mining/mine_turfs.dm [hunk 1]
```diff
@@ -129,7 +129,7 @@
 /turf/simulated/mineral/random/low_chance
 	icon_state = "rock_lowchance"
 	mineralChance = 5
-	mineralSpawnChanceList = list("Uranium" = 5, "Diamond" = 2, "Gold" = 5, "Silver" = 10, "Plasma" = 15, "Iron" = 35, "Gibtonite" = 1)
+	mineralSpawnChanceList = list("Uranium" = 5/*, "Diamond" = 2*/, "Gold" = 5, "Silver" = 10, "Plasma" = 15, "Iron" = 40, "Gibtonite" = 1)
```
Comment (Iamgoofball) [inline code/modules/mining/mine_turfs.dm]: i dunno about this man

Comment (ghost) [inline code/modules/mining/mine_turfs.dm]: It seems silly to have diamond on the east rock at all but I can revert it if you really want it to be #rare as fuck

**Comment by Atskadan** [comment]
it still seems low
only diamond, plasma, and iron are changed
and only by increments of 5

**Comment by ghost** [comment]
now there's a 20% chance of a mineral turf containing a mineral, compared to the old 10%

this is the 'general' drop rate so there should be a notable increase in minerals around now

**Comment by GunHog** [comment]
Yaaaaaaay! :+1:

**Comment by ghost** [comment]
It has a lot to do with how the map is layed out now because the old way minerals works was a little wonky but this should be close if not better than old mining but I really need to know if miners are coming back with three full stacks of diamond after 10 minutes once this has been tested. Looking at it in game it looks pretty tame, but you never know.

**Comment by ghost** [comment]
Merge this for confirmed fun

**Pull request merged**